department-of-veterans-affairs/va.gov-cms
# VACMS-2855 3008 3063

## Description
See #2855, #3008, #3063 

## Testing done
Visual, Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/94861771-0e3b1500-0406-11eb-9a64-f7ed36e8f98f.png)
![image](https://user-images.githubusercontent.com/2404547/94861790-13985f80-0406-11eb-8223-bc402d3f98d4.png)
![image](https://user-images.githubusercontent.com/2404547/94861802-198e4080-0406-11eb-8cee-36912f81d85b.png)
![image](https://user-images.githubusercontent.com/2404547/94861829-1dba5e00-0406-11eb-8772-66c14f237ef7.png)

## QA steps
Probably best to have @kevwalsh poke around manually. A good node for testing: `/node/365/edit`. Here you can ensure all field labels / help text / required fields have been set for service locations, that all phone paragraph instances toggle the extension field appropriately, and the extension field appears inline.
